### PR TITLE
[M] LP: Build Registration Form Components

### PR DESCRIPTION
**Pull request by jose-m-f**
https://trello.com/c/CfvsDQAv

### CODE REVIEW

File: src/components/login/AccountForm.jsx [hunk 1]
```diff
@@ -1,7 +1,22 @@
...
+        Last Name:
+        <input type="text" name="lastName" value={formData.lastName} onChange={handleChange} placeholder="Dieudonnée" required />
+      </label>
+      <label>
+        Email:
+        <input type="email" name="email" value={formData.email} onChange={handleChange} placeholder="melvin.dieudonnee@gmail.com" required />
+      </label>
+      <label>
+        Phone Number:
+        <input type="tel" name="phoneNumber" value={formData.phoneNumber} onChange={handleChange} placeholder="123-456-7890" pattern="[0-9]{3}-[0-9]{3}-[0-9]{4}" required />
```
Comment (clee4037) [inline src/components/login/AccountForm.jsx:18]: nice work with the phone number validation

File: src/components/login/GarageForm.jsx [hunk 2]
```diff
@@ -1,7 +1,77 @@
-const GarageForm = () => {
+const GarageForm = ({ formData, handleChange }) => {
+  const states = ['AL','AK','AZ','AR','CA','CO','CT','DE','FL','GA','HI','ID','IL','IN','IA','KS','KY','LA','ME','MD','MA','MI','MN','MS','MO','MT','NE','NV','NH','NJ','NM','NY','NC','ND','OH','OK','OR','PA','RI','SC','SD','TN','TX','UT','VT','VA','WA','WV','WI','WY']
```
Comment (clee4037) [inline src/components/login/GarageForm.jsx:2]: I have all the states in the state.js file in the lib directory. Feel free to use it!

File: src/components/login/GarageForm.jsx [hunk 3]
```diff
@@ -1,7 +1,77 @@
...
+      <label>
+        Opening Hour:
+        <select
+          name="garageOpeningHour"
+          value={formData.garageOpeningHour}
+          onChange={handleChange}
+          required
+        >
+          {Array.from({ length: 24 }, (_, i) => i).map((hour) => {
+            const displayHour = hour === 0 ? 12 : hour > 12 ? hour - 12 : hour;
```
Comment (clee4037) [inline src/components/login/GarageForm.jsx:38]: FYI, there's a convertTime function in lib/timeSlotUtil.js

**Review by clee4037 — APPROVED** [review]
Very clean and well structured. Please use convertTime function and states data in the lib directory!

File: src/components/login/GarageForm.jsx
Comment (bzw24) [inline src/components/login/GarageForm.jsx:1]: Think Christian/Steven already have the states array that you can use in your module under lib/states.js

**Review by bzw24 — APPROVED** [review]
Feel free to use the lib/states.js, since other modules seems to be using it as well

**Pull request merged**